htmresearch/algorithms/column_pooler.py

```diff
@@ -273,25 +280,68 @@ def _computeInferenceMode(self, feedforwardInput, lateralInputs):
         lateralInput, self.connectedPermanenceDistal)
       numActiveSegmentsByCell[overlaps >= self.activationThresholdDistal] += 1
 
-    # Activate some of the feedforward supported cells
-    minNumActiveCells = self.sdrSize / 2
-    chosenCells = self._chooseCells(feedforwardSupportedCells,
-                                    minNumActiveCells, numActiveSegmentsByCell)
 
-    # If necessary, activate some of the previously active cells
-    if len(chosenCells) < minNumActiveCells:
-      remaining = numpy.setdiff1d(prevActiveCells, feedforwardSupportedCells)
-      remaining = remaining[numActiveSegmentsByCell[remaining] > 0]
+    chosenCells = []
+    minNumActiveCells =  30 #self.sdrSize / 2
+
+
+    # # This is the old method. It relies heavily on the chooseCells function, which is
```

Comment:
Lot of commented out code.  If the changes look reasonable and we get the same results on our convergence scripts I would rather we just remove the commented out parts.  @mrcslws